src/tests/functional/plugin/gpu/remote_blob_tests/cldnn_remote_blob_tests.cpp

```diff
@@ -98,6 +99,64 @@ TEST_P(RemoteBlob_Test, smoke_canInputUserBlob) {
     }
 }
 
+TEST_P(RemoteBlob_Test, smoke_canUseRemoteBlobSimultaneously) {
+#if defined(ANDROID)
+    GTEST_SKIP();
+#endif
+    const int batch = 2;
+    const int channels = 3;
+    const int height = 512;
+    const int width = 512;
+    const size_t img_size = batch * channels * height * width;
+    cl_int err;
+
+    const InferenceEngine::TensorDesc tensor_desc{InferenceEngine::Precision::U8,
```

Comment:
Do you plan to migrate these tests to OV2.0?

Reply:
This test is specific to legacy API. New API doesn't allow to lock/unclock remote tensors.